packages/fluentui/docs/src/views/IntegrateCustomComponents.tsx

```diff
@@ -271,7 +271,7 @@ export default () => (
 
     <GuidesNavigationFooter
       previous={{ name: 'Layout', url: 'layout' }}
-      next={{ name: 'Styles Overrides', url: 'styles-overrides' }}
+      next={{ name: 'Performance', url: 'performance' }}
```

Comment:
sanity check: it's for this right ? 

![image](https://user-images.githubusercontent.com/20744592/112128546-705b2280-8bbe-11eb-96e0-c402217681bf.png)

Reply:
Yup